AndroidTest/app/src/main/java/cloudant/com/androidtest/MyActivity.java

```diff
@@ -15,112 +15,19 @@
 package cloudant.com.androidtest;
```

Comment:
`setTestProperties` is no longer called, I removed it in 69ea2c8 where I also added the comment for what the class is for